### PR TITLE
Updating docs to show how to use the same local version and binary packages for Windows servers

### PR DESCRIPTION
**Pull request by ntakru**

### CODE REVIEW

**Comment by ntakru** [comment]
@garyericson @extclrcommitters@service.microsoft.com @ivannp

**Review by garyericson — APPROVED** [review]
I haven't verified the code itself (I assume the other reviewers will do that), but the text looks good. Thanks for the update.

**Review by guitarnana — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -28,9 +28,22 @@ Download the latest release from https://github.com/microsoft/sqlmlutils/release
 
 Windows:
 
+To obtain the version of R your server is currently using, please use this query:
+```
```
Comment (ivannp) [inline README.md:32]: Can we change the forrmatting a bit:

```
EXEC sp_execute_external_script
    @language = N'R',
    @script = N'
        v = R.version
        OutputDataSet = data.frame(rversion=paste0(v$major, ".", v$minor))',
    @input_data_1 = N'select 1'
WITH RESULT SETS ((rversion varchar(max)));
```

**Review by ivannp — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ntakru) [reply README.md:32]: I updated the formatting right now. Thank you.

**Pull request merged**